bbb_broker.php

```diff
@@ -665,11 +666,17 @@ function bigbluebuttonbn_broker_recording_ready($params, $bigbluebuttonbn) {
     }
     // Sends the messages.
     try {
-        bigbluebuttonbn_send_notification_recording_ready($bigbluebuttonbn);
+        // That if was for trying to workaround the several emails sent for one record ready
```

Comment:
The comment should be something more specific, and perhaps with different wording. Instead of
        // That if was for trying to workaround the several emails sent for one record ready

Use something like
// Workaround for CONTRIB-7438. We make sure messages are send only once.